### ISSUE TITLE
Test jarchivelib on Windows systems

### ISSUE THREAD

**Issue by thrau** [thrau/jarchivelib#24]
Perhaps that should be done for the first major release.

also test #42

**Comment by lamirand-g** [comment]
I have issues under windows. The archive created doesn't contain any directory. In fact there are replace by a file called "parent_directory\file".

**Comment by thrau** [comment]
yes there is no windows support as of now. patches for windows support would be greatly appreciated
